clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/CreateServerGroupAtomicOperationSpec.groovy

```diff
@@ -129,6 +134,10 @@ class CreateServerGroupAtomicOperationSpec extends CommonAtomicOperation {
     def result = operation.operate([])
 
     then:
+    ecs.listAccountSettings(_) >> new ListAccountSettingsResult().withSettings(
+      new Setting(name: SettingName.TaskLongArnFormat, value: "enabled"),
+      new Setting(name: SettingName.ServiceLongArnFormat, value: "enabled")
+    )
```

Comment:
Can you also add tests to account for only Task or Service long arns being enabled?